aces/Loris
# [My Preferences] resurrected subpage help text

Created a help text for the My Preferences sub-page of the User Accounts module.

#### Testing instructions 

1. edit copy if necessary
2. see screenshot for rendering in LORIS

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576
* https://github.com/aces/Loris/issues/5727

![Screenshot 2019-11-24 19 08 12](https://user-images.githubusercontent.com/37813606/69504045-c7c78080-0eed-11ea-8fa5-381f38f702bb.png)
